vhost_user_fs/src/sandbox.rs

```diff
@@ -249,6 +256,22 @@ impl Sandbox {
         Ok(())
     }
 
+    /// Sets the limit of open files to the max possible.
+    fn setup_nofile_rlimit(&self) -> Result<(), Error> {
+        // /proc/sys/fs/nr_open is a sysctl file that shows the maximum number
+        // of file-handles a process can allocate.
+        let path = "/proc/sys/fs/nr_open";
+        let limit_str = fs::read_to_string(path).map_err(|e| Error::ReadProc(e))?;
+        let limit = limit_str
+            .trim()
+            .parse()
+            .map_err(|e| Error::InvalidNrOpen(e))?;
+
+        setrlimit(Resource::NOFILE, limit, limit).map_err(|e| Error::SetRlimit(e))?;
```

Comment:
I'm totally fine with the patch overall, but I'd like to avoid the introduction of the new crate `rlimit`. Using `libc` should be enough, and long term, I think we could implement both `getrlimit()` and `setrlimit()` abstractions as part of the `vmm-sys-util` crate in Rust-VMM.

Reply:
Replaced the `rlimit` crate with a `libc::setrlimit` call.